.vscode/tasks.json

```diff
@@ -7,22 +7,23 @@
 // ${cwd}: the current working directory of the spawned process
 // A task runner that calls a custom npm script that compiles the extension.
 {
-	"version": "0.1.0",
-	// we want to run npm
-	"command": "npm",
-	// the command is a shell script
-	"isShellCommand": true,
-	// show the output window only if unrecognized errors occur.
-	"showOutput": "silent",
-	// we run the custom script "compile" as defined in package.json
-	"args": [
-		"run",
-		"compile",
-		"--loglevel",
-		"silent"
-	],
-	// The tsc compiler is started in watching mode
-	"isBackground": true,
-	// use the standard tsc in watch mode problem matcher to find compile problems in the output.
-	"problemMatcher": "$tsc-watch"
-}
\ No newline at end of file
+	"version": "2.0.0",
+	"task": [
+		{
+			"label": "build",
```

Comment:
This label needs to match the "preLaunchTask" in `.vscode/launch.json`. I would change those to "build".

Otherwise you see this when you start debugging:
![screen shot 2018-10-19 at 10 06 54 am](https://user-images.githubusercontent.com/11282622/47233449-a3fb4800-d387-11e8-9f20-7cd0eef361f9.png)

Reply:
I changed it to "Compile". Is it OK?

Reply:
I'm fine with compile, but the important part is that it needs to match whatever the value for "preLaunchTask" is in `.vscode\launch.json`

https://github.com/Microsoft/vscode-cosmosdb/blob/master/.vscode/launch.json#L18

Reply:
Oh sorry! It is a mistake. I fixed it. Thanks.